### PR TITLE
Attach agreement letter file to acceptance mails

### PR DESCRIPTION
**Pull request by tom95**

### CODE REVIEW

**Review by aloeser — APPROVED** [review]
Worked for me locally.

File: app/models/email.rb [hunk 1]
```diff
@@ -41,6 +41,13 @@ def persisted?
 
   private
 
+  def get_agreement_letter_attachment
+    {
+      name: (I18n.t 'emails.agreement_letter_attachment'),
+      content: File.read(Rails.root.join('storage', 'agreement_letters', 'real_agreement_letter.pdf'))
```
Comment (aloeser) [inline app/models/email.rb]: I'd suggest to read the path from the config, either in this PR, or later (we need a technical debt issue then)

File: config/application.rb [hunk 2]
```diff
@@ -31,5 +31,7 @@ class ApplicationLetter < Rails::Application
 
     config.from_address = ENV['SMTP_ADDRESS'] ? ENV['SMTP_ADDRESS'] : 'noreply@localhost'
     config.reply_to_address = config.from_address
+
+    config.empty_agreement_letter_path = File.read(Rails.root.join('storage', 'agreement_letters', 'real_agreement_letter.pdf'))
```
Comment (cmfcmf) [inline config/application.rb]: I think you should remove the path suffix, because it Really is the content, not the path.

Comment (tom95) [reply config/application.rb]: @cmfcmf fun fact: I meant to only put the path :P I'll change it so that it won't be kept in memory

**Comment by cmfcmf** [comment]
Even better!

Tom Beckmann <notifications@github.com> schrieb am Do., 9. Feb. 2017, 08:43:

**Pull request merged**